packages/vite-plugin-cloudflare/package.json

```diff
@@ -69,7 +69,7 @@
 	},
 	"peerDependencies": {
 		"vite": "^6.1.0 || ^7.0.0",
-		"wrangler": "workspace:*"
+		"wrangler": "workspace:^"
```

Comment:
Just so I'm clear in my head, this will change the published package.json from:

```
  "wrangler": "4.1.0"
```
to
```
  "wrangler": "^4.1.0"
```
?